src/service/video-manager-impl.js

```diff
@@ -755,11 +876,64 @@ class VideoEntry {
    *
    * @private
    */
-  unDockVideo_() {
+  finishDocking_() {
     // Restore the video inline
     this.internalElement_.classList.remove(DOCK_CLASS);
     this.internalElement_.setAttribute('style', '');
+    this.dockingState_ = DockingStates.INLINE;
     this.video.showControls();
```

Comment:
I don't think this is that easy. If we don't show controls, users won't be able to interact with the video after it returned to its original position. This needs to be delayed until the user interacts again with the video (sort of like what we do with autoplay).